assets/base/i18n/en.json

```diff
@@ -991,11 +991,17 @@
   "channel_header.webrtc.call": "Start Video Call",
   "channel_header.webrtc.unavailable": "New call unavailable until your existing call ends",
   "channel_info.about": "About",
+  "channel_info.add_members": "Add Members",
```

Comment:
For these, can you look to see if there's a suitable existing strings that can be used for most of these? Like this one could use "chanel_header.addMembers" (which I now realize is misspelled). We're trying to share localization strings between the web and RN apps wherever possible. Everything that only exists in the native app should then be prefixed with "mobile."